server/src/uuid/model/entity/mod.rs

```diff
@@ -915,10 +915,12 @@ impl Entity {
                 })
             })
             .transpose()?
-            .map(|date| DateTime::from(date.with_timezone(&Utc)));
+            .map(|date| date.with_timezone(&Berlin).to_string());
 
         Ok(sqlx::query!(
             r#"
+                SELECT *
+                FROM (
```

Comment:
That does not work because we have to limit before we order. Otherwise we get the last ? deleted entities.

Reply:
I'll merge it in order that we remove this blocker. 
@kulla a refactor is welcome

Reply:
> That does not work because we have to limit before we order. Otherwise we get the last ? deleted entities.

Thx, I haven't seen it... 

> @kulla a later refactor is welcome

:+1: